### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by iprankurpandey**

### CODE REVIEW

File: Homepage.css [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+.third-catagory{
+    background-image: url('https://i.ibb.co/Zxxg78R/icons8-api-100.png');
+}
+.fourth-catagory{
+    background-image: url('https://i.ibb.co/tHM0CvL/dsa.png');
+}
+.box{
+    width: 15rem;
+    height: 15rem;
+    background-color: rgb(255, 255, 255);
```
Comment (sri-rishi) [inline Homepage.css:77]: use variable color naming

File: Homepage.css [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+.fifth-brand{
+    background-image: url('https://i.ibb.co/tY8G9Hs/icons8-python-480-1.png');
+}
+.sixth-brand{
+    background-image: url('https://i.ibb.co/88Lb364/icons8-git-480.png');
+}
+
+.footer-items{
+    display: flex;
+    /* justify-content: center; */
```
Comment (sri-rishi) [inline Homepage.css:141]: remove comments

File: Homepage.html [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>HomePage</title>
+    <link rel="stylesheet" href="/Homepage.css">
+</head>
+
+<body>
+    <div class="navigation-menu">
+        
+        <div class="left-side-panel" title="menu-item">
+            <div class="menu"> <span class="material-icons mtrlshop">
```
Comment (sri-rishi) [inline Homepage.html:16]: use semantic tag nav for navigation

**Review by sri-rishi — CHANGES_REQUESTED** [review]
rest look good to me

File: Homepage.css [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+    background-repeat: no-repeat;
+    background-size: cover; 
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    max-width: 100%;
+}
+.head-text{
+    font-size: 2.5rem;
+    color: rgb(248, 63, 63);
```
Comment (manish-GH) [inline Homepage.css:21]: use variables for colors

File: Homepage.css [hunk 5]
```diff
@@ -0,0 +1,171 @@
...
+    font-weight: 800;
+}
+.mtext{
+    font-size: 3rem;
+    color: rgb(99, 98, 96);
```
Comment (manish-GH) [inline Homepage.css:26]: use variable

File: Homepage.css [hunk 6]
```diff
@@ -0,0 +1,171 @@
...
+    font-weight: 600;
+}
+.logo-text{
+    font-size: 2rem;
+    color: rgb(247, 2, 96);
```
Comment (manish-GH) [inline Homepage.css:31]: use variable

File: Homepage.css [hunk 7]
```diff
@@ -0,0 +1,171 @@
...
+    font-weight: 500;
+}
+.subtext-image{
+    font-size: 1.5rem;
+    color: rgb(248, 2, 166);
```
Comment (manish-GH) [inline Homepage.css:36]: use variable

File: Homepage.css (hunk 1 above)
Comment (manish-GH) [inline Homepage.css:77]: use variable

File: Homepage.css [hunk 8]
```diff
@@ -0,0 +1,171 @@
...
+    text-align: center;
+    margin: 3rem;
+    text-decoration: underline;
+    text-decoration-color:goldenrod;
+}
+
+.brand-logo{
+    width: 8rem;
+    height: 5rem;
+    background-color: rgb(255, 255, 255);
```
Comment (manish-GH) [inline Homepage.css:105]: use variable

File: Homepage.css [hunk 9]
```diff
@@ -0,0 +1,171 @@
...
+   bottom: 0;
+   width: 100%
+}
+.social-media-links{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: row;
+    height: 2rem;
+    background-color: silver;
```
Comment (manish-GH) [inline Homepage.css:158]: use variable

File: Homepage.css [hunk 10]
```diff
@@ -0,0 +1,171 @@
...
+    background-image: url('https://i.ibb.co/88Lb364/icons8-git-480.png');
+}
+
+.footer-items{
+    display: flex;
+    /* justify-content: center; */
+    align-items: center;
+    flex-direction: column;
+    height:auto;
+    background-color: silver;
```
Comment (manish-GH) [inline Homepage.css:145]: use variable

File: Homepage.html [hunk 11]
```diff
@@ -0,0 +1,108 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>HomePage</title>
+    <link rel="stylesheet" href="/Homepage.css">
+</head>
+
+<body>
+    <div class="navigation-menu">
```
Comment (manish-GH) [inline Homepage.html:13]: you can use nav instead of div here

File: Homepage.html [hunk 12]
```diff
@@ -0,0 +1,108 @@
...
+                    favorite_border
+                </span></div>
+            <div class="user-account" title="your account details"><span class="material-icons mtrlshop">
+                    account_circle
+                </span>
+            </div>
+        </div>
+    </div>
+
+    <div class="main-container">
```
Comment (manish-GH) [inline Homepage.html:49]: main instead of div

**Review by manish-GH — COMMENTED** [review]
On large screens the footer doesn't reach the bottom, you could consider giving a min-height of 100vh to the body and flex-grow for your main content

**Pull request merged**